src/main/riddl/Events/Reservations/reservationTypeDefinitions.riddl

```diff
@@ -4,15 +4,14 @@ type Info is {
     range: 'TimeRange',
     title: String,
     description: String,
-    location: LocationId,
```

Comment:
Why wouldn't we allow somebody to update the location of an event?

Reply:
These are `Reservation` definitions - there is a commented out `location` field in `Event`, I can simply uncomment it if desired. Probably left it commented as a way of referencing question marks around whether to put it on `Reservation` or `Event`

Reply:
I the Reservation is the right place for location.  It is, effectively, the join from Event to Location. It would be very convenient to have location with the Event. I think that means we need to work on a projection definition.

Reply:
This begs some questions surrounding the issue of whether an Event can have more than one Reservation. 2 ways for that to happen...

One would be cancelled Reservations, but we don't have a ReservationStatus `Cancelled` - maybe we want one? Makes me wonder if they have to move back to Draft to change their Info? That seems not so great.

2nd, we do have both `Event.expectedStart/End` and `Reservation.range`, but maybe that's kind of redundant and can cause sync issues... I think it would make sense to have Event be time agnostic recurrences by removing the `expectedStart/End`. If we do this we can also make the Reservation in the Event be a list to allow something like recurrences without going all the way to the ISO spec for RecurrenceRules.

If we go with either of these ideas, it would probably be best to allow `ReservationStatus` to be specified in the query.

Reply:
Some good thinking in here Alex.  Here are a few thoughts in response:

I think that an event can have only 1 active reservation. As I read through the different states of Reservation it makes me think that we need to define them a bit better. For example, what is the difference between Reserved and Active? I realize that I was probably the one that created those states. I don't mind the idea of having a Cancelled state, but the question that I have (and I am questioning all the states for reservation at this point) is what business value would it hold? I will spend some time thinking here.

I think there is a case for different time ranges for reservation and event. For example, you may want your reservation for 4 hours before the event starts and 4 hours afterward for setup/teardown. But the Event has a start/end where the actual event takes place. I had not considered recurrence. This may need some thought and conversation. My initial reaction is to move it out of scope for now.